.github/workflows/bump-release-version.yml

```diff
@@ -0,0 +1,35 @@
+#############################################################################
+# GitHub Action to bump release version
+#
+#############################################################################
+name: "Bump release year"
+on:
+  workflow_dispatch:
+permissions:
+  contents: write
+  pull-requests: write
+jobs:
+  bump:
+    name: Bump release year
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the latest code
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Modify File
+        run: |
+          ./.ci/bump-release-version.sh
+      - name: Latest Release
+        id: RELEASE_TAG
+        run: |
+          NEW_RELEASE=$(mvn -e --no-transfer-progress -q -Dexec.executable='echo' \
```

Comment:
New version should be taken from tag name, please review sequence diagram

Reply:
Changes Done